test/OpenTelemetry.Shims.OpenTracing.Tests/SpanBuilderShimTests.cs

```diff
@@ -236,12 +245,13 @@ public void WithTag_KeyIsNullStringValue()
         var tracer = TracerProvider.Default.GetTracer(TracerName);
         var shim = new SpanBuilderShim(tracer, "foo");
 
-        shim.WithTag((string)null, "unused");
+        shim.WithTag((string)null!, "unused");
```

Comment:
This seems wrong to me, can you help me understand why you made this change?

Keys shouldn't be null. My instinct expects that this should throw an exception similar to calls above `        Assert.Throws<ArgumentNullException>(() => new SpanBuilderShim(null!, "foo"));`

Reply:
Thinking about this some more, looks like the previous implementation would have silently dropped the KVP. 
I think your change here is fine.

Reply:
I agree with your second message. Resolvig.